packages/docs-app/src/styles/_examples.scss

```diff
@@ -587,7 +587,8 @@ $docs-hotkey-piano-height: 510px;
 }
 
 #{example("ButtonGroup")},
-#{example("ButtonGroupPopover")} {
+#{example("ButtonGroupPopover")},
+#{example("Drawer")} {
   .docs-example > * {
     margin: 0;
```

Comment:
Removing the margin to prevent the button from jumping:
![drawer-margin](https://github.com/palantir/blueprint/assets/4532785/cfb05080-3cd4-43d4-9bf1-af7d76d05636)